docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/classrooms_core.module

```diff
@@ -215,7 +259,7 @@ function classrooms_core_preprocess_layout(&$variables) {
         break;
 
       case 'layout_twocol':
-        if (isset($variables['region_attributes']['second'])) {
+        if (isset($variables['region_attributes']['second']) && ($variables["content"]["#settings"]["label"] == "Map")) {
```

Comment:
```suggestion
        if (isset($variables['region_attributes']['second']) && ($variables['content']['#settings']['label'] === 'Map')) {
```
Fix for single quotes and using the identical operator instead the equals operator.